src/sparseml/keras/__init__.py

```diff
@@ -24,9 +24,23 @@
 
     version = [int(v) for v in tensorflow.__version__.split(".")]
     if version[0] != 2 or version[1] < 2:
-        raise Exception
+        raise RuntimeError("TensorFlow >= 2.2 is required, found {}".format(version))
 except:
     raise RuntimeError(
-        "Unable to import tensorflow. tensorflow>=2.2 is required"
+        "Unable to import tensorflow. TensorFlow>=2.2 is required"
         " to use sparseml.keras."
     )
+
+
+try:
+    import keras
+
+    v = keras.__version__
+    if v < "2.4.3":
```

Comment:
Why are we only supporting the very latest release and the very latest bug release at that? Let's expand this support, otherwise there really isn't much of a point to supporting Keras here since it was pushed to the TF package as the main flow before this

Reply:
Resolved, we can't support clone_model for previous versions. 2.4.3 is when they did the merge with tensorflow which is the reason why it works well and is an exact copy of the 2.2 implementation